kubernetes/akka-cluster.yml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
Apparently this is The New Way since Kubernetes 1.9, https://github.com/jenkinsci/kubernetes-cd-plugin/issues/50#issuecomment-415747860